README.md

```diff
@@ -1,43 +1,87 @@
-# ddf--bp--energy
+# ddf––bp––energy
+Date: 2020–11–13
 
-source: https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html
+source: https://www.bp.com/en/global/corporate/energy–economics/statistical–review–of–world–energy.html
 
 for now, only part of data(Production, Consumption) were imported. Other data (Prices, Trade) were not imported
 
+### Highlights
```

Comment:
Good that you document changes.

Please distinguish the changelog from the dataset description.

So you can have a changelog section here or a changelog.md file, separate from the description of the dataset.

Reply:
Sure, I'll change Highlights to Changelog section here, and also add a separate Changelog.md. 

I'll add Changlog section here because it might get overlooked in a separate Changelog.md, but in a Readme.md, it appears upfront so you can't miss it.